gulpfile.js

```diff
@@ -31,6 +33,10 @@ gulp.task('js:app', function (){
         ;
 });
 
+gulp.task('webserver', function() {
```

Comment:
For now, the testing and dev environment are working using Apache on linux. Could you please leave this conf in your local env?